src/etc/inc/interfaces.inc

```diff
@@ -3341,26 +3379,32 @@ function DHCP6_Config_File_Advanced($interface, $wancfg, $wanif)
             }
             $id_assoc_statement_prefix .= ";";
         }
-
-        if (is_numeric($wancfg['adv_dhcp6_prefix_interface_statement_sla_id'])) {
```

Comment:
I believe this block is for the dhcp6 interface itself. if you want to iterate for the track6 interfaces you have to do it in the same way as in basic as an additional block.

Reply:
(please no work here, I am modifying the diff on the other branch so all thoughts are helpful)